### ISSUE TITLE
Make vit-ss work without vitup

### ISSUE THREAD

**Issue by nicopado** [input-output-hk/catalyst-core#605]
this issue will unblock https://github.com/input-output-hk/catalyst-load-test/issues/5

vit-ss should work without launching it with vitup utility, so that it the loadtest can be validated against it

**Comment by minikin** [comment]
Won't do it.
